erpnext/stock/get_item_details.py

```diff
@@ -88,8 +88,20 @@ def get_item_details(args, doc=None, for_validate=False, overwrite_warehouse=Tru
 
 	update_party_blanket_order(args, out)
 
+	current_customer = args.customer
+	# Never try to find a supplier price if customer is set
+	# these Doctype
+	if (
+		args.get("doctype") == "Purchase Order"
+		or args.get("doctype") == "Purchase Receipt"
+		or args.get("doctype") == "Purchase Invoice"
+	):
```

Comment:
Use `in` keyword to check the doctype:

`if (args.get("doctype") in ["Purchase Order", "Purchase Receipt", "Purchase Invoice"]):`